pep/app/constants/cookies.ts

```diff
@@ -1,5 +1,5 @@
 export const HIDE_TOUR_COOKIE_NAME = 'showTourCookie';
-export const SESSION_COOKIE_NAME = 'ember_simple_auth-session';
+export const SESSION_COOKIE_NAME = 'ember_simple_auth-sessions';
```

Comment:
🤔

Reply:
@billdami I was seeing that really annoying dev only cookie bug.

Reply:
ugh 🤦  not that there is a better solution to this, but I hate that we have to do this, especially now that we have real users in the app, since it'll cause dropped sessions and whatnot when this is pushed to prod.